manila/share/manager.py

```diff
@@ -5642,6 +5681,20 @@ def _server_migration_complete_driver(self, context, source_share_server,
         dest_sn = self.db.share_network_get(context, dest_sn_id)
         dest_sns = self.db.share_network_subnet_get(context, dest_sns_id)
 
+        # SAPCC: Network are extended to the destination host on previous
```

Comment:
I actually choose to leave the word "allocation" out. Network allocation is created to extend the network. I tried to explain it in the next sentence "i.e. port bindings ...". Again there are mixing of terms "network allocations" and "port bindings", which could be confusing. I would change the "port bindings" to "network allocations" to be consistent.

Reply:
reading the comments again, I choose to go with "network allocations == ports", instead of "network allocations == port bindings", so network allocations are extended aka ports are extended.